modules/aws/region.go

```diff
@@ -20,6 +20,35 @@ const regionOverrideEnvVarName = "TERRATEST_REGION"
 // this region as a default.
 const defaultRegion = "us-east-1"
 
+// Reference for launch dates: https://aws.amazon.com/about-aws/global-infrastructure/
```

Comment:
Nice addition 👍

Reply:
BTW, is there any easy way to add a yearly reminder to update this? Something in GitHub? Calendar invite?

Reply:
Set a reminder in slack for now. Used 1/23 every year.